src/routes/Home.jsx

```diff
@@ -1,11 +1,5 @@
-import Searchbar from '../components/Searchbar';
-
 const Home = () => {
-  return (
-    <section>
-      <Searchbar />
-    </section>
-  );
+  return <section />;
```

Comment:
I wasn't sure if we want to show the searchbar in login and add pages too, but this seems legit that the user can search terms in any page. But just keep `<section></section>` because we will keep adding components inside Home.